staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation.go

```diff
@@ -160,6 +160,10 @@ func (s *Validator) validateExpressions(fldPath *field.Path, sts *schema.Structu
 			continue // todo: wire oldObj parameter
 		}
 		evalResult, evalDetails, err := compiled.Program.Eval(activation)
+		if err != nil {
+			errs = append(errs, field.Invalid(fldPath, obj, fmt.Sprintf("%v evaluating rule: %v", err, ruleErrorString(rule))))
+			continue
```

Comment:
we still want to handle evalDetails first so we count the cost of the evaluation

Reply:
With `continue`, the evalDetails will be handled even an error was caught here right?

Reply:
no, continue jumps back to the beginning of the for-loop, skipping evalDetails handling

Reply:
So sorry somehow my brain stuck

Reply:
> So sorry somehow my brain stuck

welcome to the club... I asked [this question](https://github.com/kubernetes/kubernetes/pull/108419#discussion_r827208071) earlier today, apparently forgetting how if statements work 🙄